float32 load                    # processor load from 0 to 1
float32 ram_usage		# RAM usage from 0 to 1
